### DISCUSSION TITLE
Unable to Apply Default Styles of ShadCN in a WXT Project

### DISCUSSION THREAD

**Discussion by joisun** [wxt-dev/wxt#1249] [Q&A] [answered]
Hello WXT team,  

I am trying to integrate ShadCN into my WXT project following the steps outlined in the [[ShadCN Manual Installation Guide](https://ui.shadcn.com/docs/installation/manual)](https://ui.shadcn.com/docs/installation/manual). However, the default styles for ShadCN components are not displaying as expected.  

![image](https://github.com/user-attachments/assets/6416479b-6a01-426c-a2b9-c258c79a9d77)

I have verified that:  
- TailwindCSS is installed and configured correctly.  
- The `tailwind.config.js` includes the necessary content paths.  
- The global CSS file (`globals.css`) is properly imported in the project.  

Despite this, the ShadCN components are rendered without their default styles.  

Could this be related to the way WXT handles CSS, or is there a specific configuration needed to make TailwindCSS and ShadCN work together in a WXT environment? Any guidance or troubleshooting steps would be greatly appreciated.  

Thank you for your assistance!  

And Here is my repo: https://github.com/joisun/Vocabify
You can also check the component on:  https://stackblitz.com/~/github.com/joisun/Vocabify?file=entrypoints/sidepanel/App.tsx

Please tell me if I should provide more related information.
Best regards, and thanks again.

**Comment by aiktb** [comment]
```diff
-  content: [ './components/**/*.{html, ts, tsx}'],
+  content: [ './components/**/*.{html,ts,tsx}'],
```

**Reply by joisun** [reply, marked as answer]
Thanks a lot, It works 😂
